app/controllers/v0/institutions_controller.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+module V0
+  class InstitutionsController < ApplicationController
+    skip_before_action :authenticate
+
+    # def index
+    #   render json: [Institution.all.limit(3).sample]
+    # end
+
+    # e.g. /v0/gibct/institutions/autocomplete?term=harv
+    #
+    def autocomplete
+      search_term = params[:term]
+      render json: Institution.autocomplete(search_term)
+    end
+
+    # e.g. /v0/gibct/institutions/profile?facility_code=31800121
+    #
+    def profile
```

Comment:
Could this be implemented as a `show` action with a route like `/v0/gibct/institutions/31800121`? Not sure how institutions will be accessed, but if it is only by facility code might make sense to map that to the identifier in routes